simulation.py

```diff
@@ -0,0 +1,73 @@
+"""
+A function to simulate alpha
+"""
+
+import copy
+import importlib
+import sys
+from thousandaire.constants import TRADING_CONFIG, DATA_LIST_ALL
+from thousandaire.data_loader import DataLoader
+from thousandaire.simulator import Simulator
+
+DATASET = 'price_dataset'
+
+def initialize():
+    """
+    Initialize data we get.
+    """
+    raw_data = DataLoader(DATA_LIST_ALL).get_all()
+    workdays = raw_data['workdays']
+    invest_data = {index : value
+                   for index, value in raw_data.items() if index != 'workdays'}
+    return_data = {}
+    for key, value in workdays.items():
+        return_data[key] = copy.deepcopy(invest_data)
+        for dataset in return_data[key].values():
+            dataset.set_workdays(value)
+    #Notice, this is a little different from other dataset in return_data.
```

Comment:
I think this line is redundant. All operations show that `workdays` is different from others.